.github/CODEOWNERS

```diff
@@ -314,6 +314,8 @@
 /core/jobs*.py @seanlip
 /core/platform/ @seanlip
 /core/templates/dev/head/App*.ts @ankita240796
+/core/templates/dev/head/app.constants.ts @ankita240796
+/core/templates/dev/head/pages/interaction-specs.contants.ts @ankita240796
```

Comment:
@seanlip I am not sure about this. Could you confirm?

Reply:
I would say @vojtechjelinek (what do you think, @vojtechjelinek)?